src/select.rs

```diff
@@ -343,8 +362,10 @@ mod tests {
         use common::Literal;
 
         let qstring = "SELECT NULL, 1, \"foo\", CURRENT_TIME FROM users;";
+        // TODO: doesn't support selecting literals without a FROM clause, which is still valid SQL
+        //        let qstring = "SELECT NULL, 1, \"foo\";";
```

Comment:
Good point!  
nested_selection could attempt to parse with something more specific than `field_definition_expr` like `field_definition_literals`(hypothetical) which would parse without the FROM (or FROM as optional).   

Worth adding a future-work issue?

Reply:
https://github.com/ms705/nom-sql/issues/56